redirects.json

```diff
@@ -1087,7 +1087,7 @@
   },
   {
     "source":"/cli/graphql/troubleshooting/",
-    "target":"/javascript/build-a-backend/graphqlapi/troubleshooting/",
+    "target":"/javascript/tools/cli-legacy/key-directive/#troubleshooting",
```

Comment:
The content from the [source link](https://docs.amplify.aws/cli/graphql/troubleshooting/) is here